jumpscale/packages/chatflows/frontend/App.vue

```diff
@@ -105,6 +105,7 @@
         end: false,
         menu: false,
         chat: CHAT,
+        title: null,
```

Comment:
Can we have a default here instead of null?

Reply:
We have a default title in the base class
https://github.com/threefoldtech/js-sdk/blob/6f0932e4a1d4a9f74df611c613a47e2515ffcd46/jumpscale/sals/chatflows/chatflows.py#L83